.idea/copyright/agpl3.xml

```diff
@@ -0,0 +1,6 @@
+<component name="CopyrightManager">
+  <copyright>
+    <option name="notice" value="Copyright (c) &amp;#36;today.year The Ontario Institute for Cancer Research. All rights reserved&#10;&#10;This program and the accompanying materials are made available under the terms of the GNU Affero General Public License v3.0.&#10;You should have received a copy of the GNU Affero General Public License along with&#10;this program. If not, see &lt;http://www.gnu.org/licenses/&gt;.&#10;&#10;THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS &quot;AS IS&quot; AND ANY                           &#10;EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES                          &#10;OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT                           &#10;SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,                                &#10;INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED                          &#10;TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;                               &#10;OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER                              &#10;IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN                         &#10;ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.&#10; &#10;" />
+    <option name="myName" value="agpl3" />
+  </copyright>
+</component>
```

Comment:
should we just gitignore `.idea` directory?

Reply:
i check this portion on purpose, because we will keep updating the licenses in future and .idea IDEs are the easiest way, some IDE settings are meant to be shared with other devs